src/librbd/internal.cc

```diff
@@ -68,6 +68,11 @@ namespace librbd {
     return image_name + RBD_SUFFIX;
   }
 
+  const string object_map_name(const string &image_id)
+  {
+    return RBD_OBJECT_MAP_PREFIX + image_id;
```

Comment:
I do think we should have a command to add and remove objectmap & exclusive locking to an image, so you can use it with existing images, or disable it for use with kernel clients